docs/using-kontena/volumes.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Volume Management
+---
+
+# Volumes
+
+Kontena provides capability to manage persistent data using volumes.
+
+Volumes are supported as first-class citizens in both Kontena Master API and on stack yaml definitions.
+
+Currently Kontena does not automatically configure any volume drivers within the grid so user must configure those themselves.
+
+
+## Managing volumes with CLI
+
+User can manage volumes with Kontena CLI with `kontena volume xyz` commands
```

Comment:
Volumes can be managed from the Kontena CLI using the `kontena volume xyz` commands.